src/stores/Edit.js

```diff
@@ -1,10 +1,12 @@
 import {action, flow, observable, toJS} from "mobx";
 import {diff} from "deep-object-diff";
 import {SortEntries} from "../utils/Utils";
+import FrameAccurateVideo from "../utils/FrameAccurateVideo";
 
 class EditStore {
   @observable saving = false;
   @observable saveFailed = false;
+  @observable clipChangeOfferings = {};
```

Comment:
what is the reason that the offerings to save to are in the store? this seems like component level state that would be passed through to the save method? is this actually referenced/used across disparate components?

Reply:
It's not used across components but there's a bit of nesting between the Save component and the component that captures the offerings selected. So it was more to avoid passing that functionality down through a few components

Reply:
I think that since it's still component state, it should be in the components, not the store. It's fine to define it as state in the root component and pass the value/modifier through the children

Reply:
Fair enough. I'll make the change

Reply:
Ok updated